<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDM - Parallel Download Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item>
        <property name="text">
         <string>All downloads</string>
        </property>
        <item>
         <property name="text">
          <string>Compressed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Downloads</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Programs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Unfinished</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Finished</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Queues</string>
        </property>
       </item>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="downloadTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>28</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>28</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Segments</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="currentTaskLabel">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="segProgressBarFrame">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="segProgressTableView">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DashLine</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>28</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>28</number>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddURL"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSort_by"/>
    <addaction name="actionHide_sidebar"/>
   </widget>
   <widget class="QMenu" name="menuDownloads">
    <property name="title">
     <string>Downloads</string>
    </property>
    <widget class="QMenu" name="menuQueue">
     <property name="title">
      <string>Queue</string>
     </property>
     <addaction name="actionStop_queue"/>
     <addaction name="actionStart_queue"/>
    </widget>
    <addaction name="actionScheduler"/>
    <addaction name="menuQueue"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSpeed_limit"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDownloads"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{spacing:25px;}</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddURL"/>
   <addaction name="actionResume"/>
   <addaction name="actionDelete"/>
   <addaction name="actionStop"/>
   <addaction name="actionScheduler"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionSort_by">
   <property name="text">
    <string>Sort by</string>
   </property>
  </action>
  <action name="actionHide_sidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide sidebar</string>
   </property>
  </action>
  <action name="actionScheduler">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_alarm_clock_48px.png</normaloff>:/icons/icons/icons8_alarm_clock_48px.png</iconset>
   </property>
   <property name="text">
    <string>Scheduler</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSpeed_limit">
   <property name="text">
    <string>Speed limit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_settings_48px.png</normaloff>:/icons/icons/icons8_settings_48px.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAddURL">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_add_48px.png</normaloff>:/icons/icons/icons8_add_48px.png</iconset>
   </property>
   <property name="text">
    <string>Add URL</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_play_button_circled_48px.png</normaloff>:/icons/icons/icons8_play_button_circled_48px.png</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_cancel_48px.png</normaloff>:/icons/icons/icons8_cancel_48px.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icons8_remove_48px.png</normaloff>:/icons/icons/icons8_remove_48px.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionStart_queue">
   <property name="text">
    <string>Start queue</string>
   </property>
  </action>
  <action name="actionStop_queue">
   <property name="text">
    <string>Stop queue</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
